core/lib/config/src/configs/en_config.rs

```diff
@@ -19,4 +19,5 @@ pub struct ENConfig {
     pub main_node_rate_limit_rps: Option<NonZeroUsize>,
 
     pub gateway_url: Option<SensitiveUrl>,
+    pub bridge_addresses_refresh_interval_sec: Option<NonZeroU64>,
```

Comment:
if a partner forgets to provide the value, I guess the consequences will be uncomfortable, right? maybe it makes sense to replace it with some sensible default constant in case the value is not provided

Reply:
@StanislavBreadless I'm merging it for now, please reply and I will adjust later if required